lab_Arrays/src/main/java/arraystasks/Main.java

```diff
@@ -0,0 +1,33 @@
+package arraystasks;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.Arrays;
+import java.util.Random;
+
+public class Main {
+    private static final Logger logger = LogManager.getLogger(Main.class);
+
+    public static void main(String[] args) {
+        CreateArray create = new CreateArray();

```

Comment:
CreateArray  -> class name should be a noun, this name sounds strange for me I would rename to something like ArrayCreator, ArrayProducer, ArrayGenerator .....